packages/react/ds/src/heading/heading.tsx

```diff
@@ -1,79 +1,51 @@
-export type HeadingAs = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
-export type HeadingSize = 'xl' | 'lg' | 'md' | 'sm' | 'xs' | '2xs';
-export type HeadingProps = {
+import { tv, type VariantProps } from 'tailwind-variants';
+import { cn } from '../cn.js';
+
+const headingVariants = tv({
```

Comment:
Could you add a test with the following properties:
```
size: 'xl',
as: 'h6`
```
That should render an `h6` tag with an `xl` style.
I'm concerned about how it prioritizes the styles.

Reply:
https://github.com/ogcio/govie-ds/commit/e0c0288d4009dcf5b22f3df88abd7257c69657d7